internal/topology/variables/cluster_variable_validation_test.go

```diff
@@ -23,17 +23,21 @@ import (
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
+var testCtx = ctrl.SetupSignalHandler()
```

Comment:
```suggestion
var ctx = ctrl.SetupSignalHandler()
```
Let's just call this only ctx, we always do in tests

Reply:
Done.